opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap_gen.py

```diff
@@ -186,6 +186,7 @@
     },
 ]
 default_instrumentations = [
+    "opentelemetry-instrumentation-asyncio==0.44b0.dev",
```

Comment:
Not sure if we want to include these in the default_instrumentations.

Reply:
Does this need to be confirmed by ocelotl?

Reply:
I checked it out. @lzchen 
If it have instruments=[] in asyncio pyproject, it will be the default_instrumentations

Asyncio is a python built-in library and doesn't have a version.
So it was developed to be instrumented by setting the name of the coroutine you want to instrument.

```bash
export OTEL_PYTHON_ASYNCIO_COROUTINE_NAMES_TO_TRACE=instrumented_coroutine
```

It is not a full instrumentation, please note.